node/hubController.js

```diff
@@ -80,8 +80,13 @@ class HubController {
     /**
      * given a specific hub info, does the onboarding given the onboardingInfo and returns the auth info
      */
-    onboard(hubId, onboardingInfo, logger) {
+    onboard(hubId, onboardingInfo, logger, flightInfo) {
```

Comment:
>flightInfo [](start = 43, length = 10)

validate that it is an object or null

Reply:
we will want to assign flight info to null if null comes in to clear stale data.